docs/user/ml/index.asciidoc

```diff
@@ -187,14 +190,20 @@ image::user/ml/images/ml-change-point-detection.png[Change point detection UI]
 
 Select a function and a metric field, then pick a date range to start detecting 
 change points in the defined range. Optionally, you can split the data by a 
-field. If the cardinality of the split field is greater than 10,000, then only 
-the first 10,000, sorted by document count, are analyzed.
-
-If a change point is detected, a chart visualizes where the change point was identified in
-the time window analyzed, making the interpretation easier. If you split the analysis by a 
-field, a separate chart is displayed for every partition with a detected change 
-point. You can view the type of change point in the chart as well as its value 
-and the time when the change happened. The corresponding `p-value` indicates how 
-extreme the change is; lower values mark more significant changes. You can use 
-the change point type selector to filter the results by specific types of change 
-points.
\ No newline at end of file
+field. If the cardinality of the split field exceeds 10,000, then only the first 
+10,000, sorted by document count, are analyzed. You can select multiple metric 
+fields and split fields to identify change points.
```

Comment:
I reckon we should emphasise that the user can configure multiple combinations of a function applied to a metric field, partitioned by the split field. With a limit of 6 configurations.